app/views/createAccount/register_assistant.scala.html

```diff
@@ -21,72 +21,94 @@
-@main_template(title = Messages("enrolment.title"), topNavigation = false) {
-
-    <div class="grid-row">
-        <div class="column-two-thirds">
-
-            <h1 class="heading-xlarge">@messages("enrolment.title")</h1>
-            <p class="form-hint">@messages("enrolment.assistant.hint")</p>
-            <h2 class="heading-medium">@messages("enrolment.assistant.organisation.details.title")</h2>
-
-                <table>
-                <tbody><tr>
-                    <th>@messages("enrolment.organisation.name.label")</th>
-                    <td>@assistantFieldData.businessName</td>
-                </tr>
-                    <tr>
-                        <th>@messages("label.postal.address")</th>
-                        <td>@Html(capitalizedAddress(assistantFieldData.businessAddress).replaceAll(", ", "<br>"))
-                        </td>
-                    </tr>
-                    <tr>
-                        <th>@messages("label.telephone.number")</th>
-                        <td>@assistantFieldData.businessPhoneNumber</td>
-                    </tr>
-                    <tr>
-                        <th>@messages("label.correspondence.email.address")</th>
-                        <td>@assistantFieldData.email</td>
-                    </tr>
-                </tbody></table>
-
-
-
-            @form(controllers.registration.routes.RegistrationController.submitAssistant()) {
-
-                <h2 class="heading-medium">@messages("label.your.details")</h2>
-
-                @includes.errorSummary(model, messagePrefix = Some("createAccount"))
-
-
-                @inputText(
-                    field = model(keys.firstName),
-                    '_label -> Messages("enrolment.firstName"),
-                    '_labelClass -> "form-label-bold",
-                    '_showConstraints -> false,
-                    'class -> "form-control",
-                    '_value -> assistantFieldData.firstName
-                )
-
-                @inputText(
-                    field = model(keys.lastName),
-                    '_label -> Messages("enrolment.lastName"),
-                    '_labelClass -> "form-label-bold",
-                    '_showConstraints -> false,
-                    'class -> "form-control",
-                    '_value -> assistantFieldData.lastName
-                )
-
-                <p>
-                    <button type="submit" id="save-and-continue" class="button">@messages("label.confirm")</button>
-                </p>
-            }
-
-        </div>
+@isAgent(selected: Boolean) = @{
+if(selected) """checked="checked""" else """ disabled="disabled" """
+}
+
+@getEnrichedErrorLinks = @{
+    model.errors.map{ error =>
+    ErrorLink(href = Some(s"#${error.key}"),
+    content = HtmlContent(s"${messages(s"label.${error.key}")} - ${messages(error.message)}") )}
+}
+
+@mainLayout(pageTitle = messages("assessments.title"), mainClass = Some("govuk-grid-column-full")) {
+
+<div class="grid-row">
```

Comment:
you don't need these 2 outer divs (same in the other pages)

Reply:
good point!

Reply:
@zahiriqbal after you have removed the 2 outer divs...do not pass "mainClass" tp mainLayout, mainLayout is already "govuk-grid-column-two-thirds" by default

Reply:
unless he needs `govuk-grid-column-full` 🤷 (as he was using both previously I'm not sure which one was actually applied)

Reply:
i think the page was previously "column-two-thirds" so I'll assume we are not looking to change that

Reply:
yeah - should keep the original 👍 (just a bit confused by Z's layout :) )